src/lib/forward.jl

```diff
@@ -101,3 +136,9 @@ forwarddiff(f, x) = f(x)
   y, J = forward_jacobian(f, x)
   return y, ȳ -> (nothing, reshape_scalar(x, J*vec_scalar(ȳ)))
 end
+
+# Second derivatives
+@adjoint ForwardDiff.derivative(f, x) = pullback(forwarddiff, x -> ForwardDiff.derivative(f, x), x)
```

Comment:
These seem like they would affect a lot more than the intended functions.

Reply:
I guess these aren't in the end used by the final version. Can add tests, or move to another PR if you prefer.